docs/src/SelectList.doc.js

```diff
@@ -20,185 +41,361 @@ card(
-    },
-    {
-      value: "cycling",
-      label: "Cycling"
-    }
-  ];
-  return (
-    <SelectList
-      id="hobby"
-      name="hobby"
-      errorMessage={ hobby === 'sailing' ? 'Sailing is not a hobby, it is a passion' : null}
-      onChange={({ value }) => setHobby(value)}
-      options={hobbies}
-      placeholder="Select hobby"
-      label="Hobby"
-      value={hobby}
-    />
-  );
-}
-    `}
-  />,
+  <MainSection name="States">
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="lg"
+        title="Default"
+        defaultCode={`<SelectList
+        id="selectlistexample"
+        onChange={() => {}}
+        options={[
+          {label:'Austin', value: 'austin'},
+          {label:'Los Angeles', value: 'la'},
+          {label:'New York', value: 'nyc'},
+          {label:'San Francisco', value: 'sf'},
+          {label:'Seattle', value: 'seattle'},
+        ]}
+        label='City'
+      />`}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        title="Focused"
+        defaultCode={`<SelectList
+        id="selectlistexample"
+        onChange={() => {}}
+        options={[
+          {label:'Austin', value: 'austin'},
+          {label:'Los Angeles', value: 'la'},
+          {label:'New York', value: 'nyc'},
+          {label:'San Francisco', value: 'sf'},
+          {label:'Seattle', value: 'seattle'},
+        ]}
+        label='City'
+        focus
+      />`}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        title="Error"
+        description={`Use \`errorMessage\` to pass in a description of the error in SelectList.`}
+        defaultCode={`<SelectList
+        id="selectlistexample"
+        onChange={() => {}}
+        options={[
+          {label:'Basketweaving', value: 'basketweaving'},
+          {label:'Bird watching', value: 'birdwatching'},
+          {label:'Comic books', value: 'comicbooks'},
+          {label:'Juggling', value: 'juggling'},
+          {label:'Sailing', value: 'sailing'},
+        ]}
+        value='sailing'
+        label='Hobby'
+        errorMessage='Sailing is not a hobby, it is a passion'
+      />`}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        title="Disabled"
+        defaultCode={`<SelectList
+        id="selectlistexample"
+        onChange={() => {}}
+        options={[
+          {label:'Austin', value: 'austin'},
+          {label:'Los Angeles', value: 'la'},
+          {label:'New York', value: 'nyc'},
+          {label:'San Francisco', value: 'sf'},
+          {label:'Seattle', value: 'seattle'},
+        ]}
+        label='City'
+        disabled={true}
+      />`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
 );
 
+card(
+  <MainSection name="Related">
+    <MainSection.Subsection
+      description={`
+**[Dropdown](/Dropdown)**
+If additional functionality is needed in the menu, such as subtext, headers or custom styling, use Dropdown.
+
+**[Typeahead](/Typeahead)**
```

Comment:
I would add Radiobutton,
If the selection list is 4 items or less, use Radiobutton.

Could Checkbox be another of these cases?

Reply:
Great callout. I'll add!